packages/tag/TagSelector.tsx

```diff
@@ -0,0 +1,88 @@
+import { h, Component, emit, prop } from 'skatejs';
+import styles from './Tag.scss';
+import { Tag } from './Tag';
+
+
+// public
+interface TagSelectorProps extends JSX.HTMLProps<HTMLElement | any> {
+  onTagChange?: Function,
+  delimiter?: string,
+  tags?: any
+}
+
+export class TagSelector extends Component<TagSelectorProps> {
+  static get is() { return 'bl-tag-selector' }
+  static get props() {
+    return {
+      tags: prop.array({
+        attribute: true
+      }),
+      delimiter: prop.string({
+        attribute: true
+      }),
+    }
+  }
+
+  tags = [];
+  delimiter = ' '; // default value is space ' '
+
+  private handleInput( event ) {
+    const lastChar = event.target.value.substr(-1);
+    const value = event.target.value.slice(0, -1).trim();
+    const isDelimiter = lastChar === this.delimiter;
+
+    if ( value && isDelimiter ) {
+      this.addTag(value);
+      event.target.value = '';
+    }
+  }
+
+  private addTag(value:string){
+    this.tags.push(value);
+    this.emitNewData( this.tags );
+  }
+
+  private removeTag(event: CustomEvent){
```

Comment:
let's be consistent with naming

removeTag --> handleTagClose

because `<Tag onTagClose={}/>`